test/f90_correct/src/maxloc14.f90

```diff
@@ -0,0 +1,16 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+! maxloc with minimum value of integer type as elements
+
+program t
+  integer, dimension(2) :: rslt, expect
+  integer, dimension(2, 2) :: m
+  m(1, :) = (/-huge(integer) - 1, -huge(integer) - 1/)
+  m(2, :) = (/-huge(integer) - 1, -huge(integer) - 1/)
```

Comment:
m = -huge(integer) - 1 would work here right?

Reply:
Yes, it would. Thanks for the suggestion.